tests/integration/tier0/non-destructive/duplicate-pkgs/test_duplicate_pkgs.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from conftest import SYSTEM_RELEASE_ENV
+from envparse import env
+
+
+DUPLICATE_PKG_URL_MAPPING = {
+    "centos-7": "https://vault.centos.org/7.4.1708/os/x86_64/Packages/python2-cryptography-1.7.2-1.el7.x86_64.rpm",
+    "oracle-7": "https://yum.oracle.com/repo/OracleLinux/OL7/latest/x86_64/getPackage/abrt-2.1.11-50.0.1.el7.x86_64.rpm",
+}
+
+DUPLICATE_PKG_URL = DUPLICATE_PKG_URL_MAPPING[SYSTEM_RELEASE_ENV]
+
+
+@pytest.fixture(scope="function")
+def install_duplicate_pkg(shell):
+    """
+    Install duplicate package
+    """
+    pkg = ""
+    pkg_was_installed = True
+
+    if SYSTEM_RELEASE_ENV == "centos-7":
+        pkg = "python2-cryptography"
+    elif SYSTEM_RELEASE_ENV == "oracle-7":
+        pkg = "abrt"
+
+    # Install `pkg` from the latest repository
+    if shell(f"rpm -q {pkg}").returncode == 1:
+        shell(f"yum install -y {pkg}")
+        pkg_was_installed = False
+
+    # Download and install duplicate package with different version
+    shell(f"curl -o duplicate-{pkg}.rpm {DUPLICATE_PKG_URL}")
+    shell(f"rpm -i --noscripts --justdb --nodeps --force duplicate-{pkg}.rpm")
+
+    assert int(shell(f"rpm -q {pkg} | wc -l ").output) >= 2
+
+    yield
+
+    # This should remove both the packages
+    shell(f"yum remove -y {pkg}")
+
+    # If the package (single one) was installed already on the system, then install it back
+    if pkg_was_installed:
+        shell(f"yum install -y {pkg}")
+
+
+@pytest.mark.test_duplicate_pkgs
+def test_list_third_party_pkgs_error(convert2rhel, install_duplicate_pkg):
+    """
+    This test verifies, that the  TASK - [Convert: List third-party packages]
+    won't fail listing packages if previously problematic third party packages are installed.
+    Installed package(s):
+    v8-devel from the epel repository
+    nodejs from the epel repository
+    """
```

Comment:
```suggestion
def test_list_third_party_pkgs_error(convert2rhel, install_duplicate_pkg):
    """
    This test verifies, that the  TASK - [Convert: List third-party packages]
    won't fail listing packages if previously problematic third party packages are installed.
    Installed package(s):
    v8-devel from the epel repository
    nodejs from the epel repository
    """
    
```
This seems to be out of place.